projects/rocprofiler-compute/CHANGELOG.md

```diff
@@ -97,6 +97,11 @@ Full documentation for ROCm Compute Profiler is available at [https://rocm.docs.
   * L1I-L2 Bandwidth
   * sL1D-L2 BW
 
+* Moved commmand-line options to top-level
+  * `--version`, `--list-metrics`, `--blocks` and `--config-dir` are now available as a top-level option.
```

Comment:
I don't see why we need to create another concept "top-level". From my understanding, the existing "General Options" is designed as a general option set applies to all levels. The simple way is just moving "--list-metrics" to "General Options".
For this specific PR, we should focus on single task "Streamline --list-metrics". Nothing to do with `--version`.
And don't touch `--blocks` and `--config-dir`